### DISCUSSION TITLE
Is there a way to exclude a particular link from being affected by a Link Note?

### DISCUSSION THREAD

**Discussion by Brettelectric** [SkepticMystic/breadcrumbs#365] [Q&A]
For example, I have an MOC note called 'Plot Ideas MOC'. Inside, there are about 100 links to other notes which are plot ideas for my book. I'm using an MOC system, so this note is like a folder containing all my plot ideas.

I want to use a Link Note to convert all these into 'down' links, like so:

BC-link-note: down

HOWEVER, there is a small paragraph at the top of this MOC which says 'This is an MOC for all my plot ideas. For scene ideas, see [[Scene Ideas MOC]]'. This link should not be a down link. It's a cross link to another MOC that has a similar function. I don't want that link to be shown on my Breadcrumbs as being below the 'Plot Ideas MOC'.

Is there any way I can exclude that one link from the effect of the Link Note?

**Comment by michaelpporter** [comment]
Shipped in v4: add `BC-list-note-exclude` to the list note's frontmatter as a list of wikilinks; any listed target is skipped (index, nested-child, and neighbour edges). Closing as resolved — thanks!
